src/actions/auger/test_auger_train.ts

```diff
@@ -0,0 +1,228 @@
+import * as chai from "chai"
+import * as httpRequest from "request-promise-native"
+import * as sinon from "sinon"
+// import * as winston from "winston"
+import * as Hub from "../../hub"
+import { Transaction } from "./poller"
+
+import { AugerTrainAction } from "./auger_train"
+
+const action = new AugerTrainAction()
```

Comment:
use `action.executeInOwnProcess = false` to make sure we are just testing your action and not the process queueing mechanisms